src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-function/GeneFunction.tsx

```diff
@@ -45,13 +48,38 @@ const tabClassNames = {
 };
 
 type Props = {
-  geneId: string;
+  gene: Gene;
   isNarrow: boolean;
-  selectedTabName: GeneFunctionTabName;
+  selectedTabName: GeneFunctionTabName | null;
   setActiveGeneFunctionTab: (tab: string) => void;
 };
 
 const GeneFunction = (props: Props) => {
+  const {
+    gene: { transcripts }
+  } = props;
+  let { selectedTabName } = props;
+
+  // Disable the Proteins tab if there are no transcripts data
```

Comment:
I am confused. Are there any genes without transcripts? Isn't transcripts in this case an empty array, which is never falsy?

Reply:
Yes, that's right. I was meant to do this check but I missed it. Thanks for spotting it.

Reply:
That's now updated 👍